lang/test/org/partiql/lang/eval/visitors/SelectStarVisitorTransformTests.kt

```diff
@@ -1,13 +1,14 @@
-package org.partiql.lang.ast.passes
+package org.partiql.lang.eval.visitors
 
 import junitparams.Parameters
 import org.junit.Test
-import org.partiql.lang.eval.visitors.GroupByItemAliasVisitorTransform
+import org.partiql.lang.ast.passes.PipelinedRewriter
+import org.partiql.lang.ast.passes.RewriterTestBase
+import org.partiql.lang.ast.passes.RewriterTransformBridge
 
-class SelectStarRewriterTests : RewriterTestBase() {
-
-    // SelectStar rewriter depends on UniqueNameMetas injected by GroupByItemAliasVisitorTransform.
-    private val rewriters = PipelinedRewriter(RewriterTransformBridge(GroupByItemAliasVisitorTransform(), ion), SelectStarRewriter())
+class SelectStarVisitorTransformTests : RewriterTestBase() {
+    // SelectStar visitor transform depends on UniqueNameMetas injected by GroupByItemAliasVisitorTransform.
+    private val rewriters = PipelinedRewriter(RewriterTransformBridge(GroupByItemAliasVisitorTransform(), ion), RewriterTransformBridge(SelectStarVisitorTransform(), ion))
```

Comment:
These tests will still use `Rewriter` and `RewriterTestBase` until `PipelinedRewriter` is migrated to PIG visitor transforms. These tests will use visitor transforms as part of the `PipelinedRewriter` migration PR.